run-test-docker-image.sh

```diff
@@ -44,16 +41,18 @@ if [ "$skip_build" = true ]; then
   echo "Skipping the build"
 else
   echo "Building image..."
+  docker system prune --force
+  git submodule update --init --recursive
   docker rmi volttron/volttron:develop --force
   docker build -t volttron/volttron:develop .
 fi
 
 
 ############ Setup and start container
 docker-compose up --detach
-echo "Configuring and starting Volttron platform (this will take some time around 5 minutes)........"
+echo "Configuring and starting Volttron platform; this will take approximately several minutes........"
```

Comment:
Where did you come up with 5 min?  Is that based upon your system?

Reply:
It was based on my system and also on the Github server. On my system, I noted the logs and saw that the bootstart.sh script usually took around 4 minutes to complete for Volttron to install the agents, shutdown, and then restart again. On the GH server, I also noticed the same length in completion time. To be safe, I put the wait time at 5 minutes to ensure that tests would not fail before the container is finished setting up.

Reply:
After some more testing, I noticed that GH Actions is inconsistent with running the test script. Sometimes the tests execute successfully, sometimes they fail because the container is not running (and that's after increasing the sleep to 6 minutes). I suspect something is going on in the GH Actions server, so I'm adding logs and rerunning the workflow to see what's happening to the container.